picard/config_upgrade.py

```diff
@@ -342,6 +343,49 @@ def upgrade_to_v2_6_0_beta_3(config):
     _s.remove("use_system_theme")
 
 
+def upgrade_to_v2_7_0_dev_1(config):
+    """Replace manually set persistent splitter settings with automated system.
+    """
+    def _check_and_add(new_key, key_map):
```

Comment:
This name isn't very explicit. I suggest `upgrade_persisted_splitter()` (or something like that?)

Also `new_key` is used in parameters and `newkey` in the loop, that's ambiguous.
What about `new_persist_key` and `old/new_splitter_key`?